waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/AbstractMetaStore.java

```diff
@@ -51,15 +51,15 @@ public abstract class AbstractMetaStore {
   private List<String> writableDatabaseWhitelist;
   private List<String> mappedDatabases;
   private @Valid List<MappedTables> mappedTables;
-  private Map<String, String> databaseNameMapping = Collections.emptyMap();
+  private transient Map<String, String> databaseNameMapping = Collections.emptyMap();
```

Comment:
why this had to change?

Reply:
@yangyuxia if you could revert or explain why this change was made I'm happy to approver once that's done,thanks!